index.html

```diff
@@ -12,5 +13,74 @@
 
 <body>
 <!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+<div class="container">
+  <header>
+    <div class="nav-container">
+      <nav>
+        <a href="services.html">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+        <a href="#"><img class="logo" src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/logo.png" alt="Great Idea! Company logo."></a>
+      </nav>
+    </div>
+  </header>
+
+  <div class="top-container">
+    <div class="title-card item">
+      <h1> Innovation<br/> On<br/>Demand<br></h1>
+      <div class="button-div">
+        <button>Get Started</button>
+      </div>  
+    </div>
+
+    <div class="circle-img"> <img src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/header-img.png" alt="Image of a code snippet.">
+    </div>
+  </div>
+  <div class="main-content-container1">
```

Comment:
```suggestion
  <main class="main-content-container1">
```
A main tag would be more semantic here